docs/pages/blog/bringing-consistency-to-material-ui-customization-apis.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Bringing consistency to Material UI customization APIs
+description: We're standardizing two key areas of the Material UI customization APIs to reduce complexity and cognitive overhead. Read on to learn what's changing.
+date: 2024-03-11T00:00:00.000Z
+authors: ['diegoandai']
+tags: ['Material UI', 'Product']
+card: true
+---
+
+The MUI Core team is working on two initiatives to standardize the Material UI API: The first applies to overriding inner elements, and the second applies to component CSS classes.
```

Comment:
```suggestion
The Material UI team is working on two initiatives to standardize the Material UI API: The first applies to overriding inner elements, and the second applies to component CSS classes.
```

I'm wondering if it makes sense to expose the "MUI Core team" concept... doesn't feel like it mean anything